modules/interactive_music/doc_classes/AudioStreamPlaybackPlaylist.xml

```diff
@@ -7,4 +7,33 @@
 	</description>
 	<tutorials>
 	</tutorials>
+	<methods>
+		<method name="change_clip">
+			<return type="void" />
+			<param index="0" name="clip_index" type="int" />
+			<description>
+				Changes to a specific clip within the playlist (by index).
+			</description>
+		</method>
+		<method name="get_current_clip_index">
+			<return type="int" />
+			<description>
+				Get the currently playing audio clip index.
+			</description>
+		</method>
+		<method name="next_clip">
+			<return type="void" />
+			<description>
+				Play the previous clip. The next random clip is played if [member AudioStreamPlaylist.shuffle] is enabled.
```

Comment:
```suggestion
				Play the previous clip. The previous random clip is played if [member AudioStreamPlaylist.shuffle] is enabled.
```

Reply:
Good catch, I somehow mixed up both next_clip() and previous_clip() texts.